R/post-body.R

```diff
@@ -1,34 +1,52 @@
 postBodyFilter <- function(req){
   handled <- req$.internal$postBodyHandled
-  if (is.null(handled) || handled != TRUE){
-    body <- paste0(req$rook.input$read_lines(), collapse = "\n")
-    charset <- getCharacterSet(req$HTTP_CONTENT_TYPE)
-    args <- parseBody(body, charset)
-    req$postBody <- body
+  if (is.null(handled) || handled != TRUE) {
+    # This will return raw bytes
+    body <- req$rook.input$read()
+    type <- req$HTTP_CONTENT_TYPE
+    args <- parseBody(body, type)
     req$args <- c(req$args, args)
+    req$rawPostBody <- body
+    if (getOption("plumber.postBody", TRUE)) {
```

Comment:
Use `isTRUE()`

Reply:
Added. Note on performance, it adds about a microsecond of execution time. I don't think it is a problem.